modules/persistent/arrow-persistent-data-structures/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'kotlin-kapt'
+
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    compile project(':arrow-annotations')
+    compile project(':arrow-core')
+    compile project(':arrow-core-data')
```

Comment:
```suggestion
```
you only need to depend on arrow-core, it will pull arrow-core-data automatically.